xsrc/test/scripts/refactor/rename-dirs.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash 
```

Comment:
> 2. The output from the bash scripts will be easier to understand if you run them with #!/bin/bash -x, so I've suggested these changes below.

All the bash scripts should be updated with -ex and the main refactor.sh calls them using the shebang instead of explicitly through bash. 
 
> 3. The scripts changed the packages of the src/it tests.  I think their packages should have stayed the same since no one ever packages test classes in jars:

This got refactored in another commit so I think this has been overcome by events. 

>  7. I ran sbt IntegrationTest/test and it failed immediately:

The reference in the documentation has been updated along with the main class reference in the templates.

>  8. I ran sbt ratCheck and it failed (the 11 licenses were the files in xsrc/test/refactor/scripts and the 5 binaries were files that weren't recognized as AL after the scripts changed them):

I added licenses where they were missing. I do not know how to fix the services files into being recognized as text. I cannot seem to find where this is set in this codebase and looking at the rat repository didn't help either. Any help here would be appreciated.

Reply:
I worked with Steve on the ratCheck issue. After digging around a bit it appears that it was getting flagged as a binary because the filename contained ".lib". Those files have been placed in the RAT exclusion list for now as there doesn't seem to be an easy way to override that behavior.